index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="index.css">
+    <!-- Template 2-->
+</head>
+
+<body>
+    <header>
+        <div class="header_container">
+            <div class="item flex_one">
+                <h1 class="name">John Doe</h1>
+            </div>
+            <div class="item flex_two">
+                <nav>
+                    <div class="menu_bar">
+                        <img src="img/Union.png" alt="Menu bar">
+                    </div>
+                </nav>
+            </div>
+        </div>
+    </header>
+    <main class="main_container">
+        <section>
+            <h1>Hey There. I’m Farida</h1>
```

Comment:
Kindly use "Crete Round" font for this section. Please check Figma template for the font size.